NAMESPACE

```diff
@@ -119,7 +119,7 @@ exportMethods(activeView, "activeView<-", blocks, browseGenome,
 export(genomeBrowsers, start, end, strand, "start<-", "end<-", width, "width<-",
        ranges, values, GenomicData, GenomicSelection,
        score, "score<-", as.data.frame, space,
-       ucscGenomes,
+       ucscGenomes, ucscTables,
```

Comment:
Please document `ucscTables`

Reply:
In which file should I add the documentation for it?
As  `ucscTables` is not part of any class.

Reply:
For now, I added it under the `UCSCTableQuery` as `ucscTables` is related to the table query interface.